modules/ztp-about-ztp-and-distributed-units-on-single-node-openshift-clusters.adoc

```diff
@@ -6,13 +6,13 @@
 [id="about-ztp-and-distributed-units-on-single-node-clusters_{context}"]
 = About ZTP and distributed units on single nodes
 
-You can install a distributed unit (DU) on a single node at scale with {rh-rhacm-first} (ACM) using the assisted installer (AI) and the policy generator with core-reduction technology enabled. The DU installation is done using zero touch provisioning (ZTP) in a disconnected environment.
+You can install a distributed unit (DU) on many OpenShift clusters at scale with {rh-rhacm-first} (ACM) using the assisted installer (AI) and the policy generator with core-reduction technology enabled. The DU installation is done using zero touch provisioning (ZTP) in a disconnected environment.
```

Comment:
This looks like it has a lot of duplication with the `about-ztp-and-distributed-units-on-openshift-clusters.adoc` module; Do we know why there are 2 of it?

Reply:
@StephenJamesSmith This looks like a dupe module. I will remove `modules/ztp-about-ztp-and-distributed-units-on-single-node-openshift-clusters.adoc`